src/components/themr.js

```diff
@@ -84,11 +84,12 @@ export default (componentName, localTheme, options = {}) => (ThemedComponent) =>
 
 function themeable(style = {}, theme) {
   if (!theme) return style
-  return [ ...Object.keys(theme), ...Object.keys(style) ].reduce((result, key) => (
-    typeof theme[key] === 'string' && style[key] && theme[key].indexOf(style[key]) === -1
-      ? { ...result, [key]: `${style[key]} ${theme[key]}` }
-      : { ...result, [key]: theme[key] || style[key] }
-  ), {})
+  return Object.keys(theme).reduce((result, key) => (
+    Object.assign(result, { [key]:
+       style[key] && theme[key].indexOf(style[key]) === -1
+       ? `${style[key]} ${theme[key]}`
+       : theme[key] || style[key]
+    })), style)
```

Comment:
Sorry, honestly I did not dig deep before merging but I wanted to give a better look before releasing. Thanks for keeping an eye on it @raveclassic!